### PR TITLE
Update to latest JSC

### PR DESCRIPTION
**Pull request by ivanbuhov**
https://github.com/NativeScript/ios-runtime/issues/416
Ready for review.

### CODE REVIEW

**Comment by ivanbuhov** [comment]
run ci

File: src/NativeScript/GlobalObject.mm [hunk 1]
```diff
@@ -216,6 +216,9 @@ static EncodedJSValue JSC_HOST_CALL collectGarbage(ExecState* execState) {
     }
 
     StringImpl* symbolName = propertyName.publicName();
+    if(symbolName == nullptr)
```
Comment (jasssonpet) [inline src/NativeScript/GlobalObject.mm]: indentation

File: src/debugging/Inspector/Inspector/NativeScript Inspector/InspectorFrontendHost.h [hunk 2]
```diff
@@ -10,6 +10,7 @@
 - (NSString*)platform;
 - (NSString*)localizedStringsURL;
 - (NSString*)debuggableType;
+- (unsigned) inspectionLevel;
```
Comment (jasssonpet) [inline src/debugging/Inspector/Inspector/NativeScript Inspector/InspectorFrontendHost.h]: indentation

File: src/debugging/Inspector/Inspector/NativeScript Inspector/InspectorFrontendHostProtocol.h [hunk 3]
```diff
@@ -6,10 +6,11 @@
 - (NSString *)platform;
 - (NSString *)localizedStringsURL;
 - (NSString *)debuggableType;
+- (unsigned) inspectionLevel;
```
Comment (jasssonpet) [inline src/debugging/Inspector/Inspector/NativeScript Inspector/InspectorFrontendHostProtocol.h]: indentation

**Comment by ivanbuhov** [comment]
Not ready for merge, yet.

**Pull request merged**